WTG.Analyzers/Analyzers/Emit/EmitAnalyzer.cs

```diff
@@ -67,7 +66,7 @@ static bool TryIdentifyEmitMethod(SemanticModel model, InvocationExpressionSynta
 		{
 			var name = ExpressionHelper.GetMethodName(invoke)?.Identifier.Text;
 
-			if (name == nameof(ILGenerator.Emit) || name == nameof(ILGenerator.EmitCall) || name == nameof(ILGenerator.EmitCalli))
```

Comment:
I'm a little concerned that paket was doing something that allowed this to compile when ILGenerator wasn't added to net standard until 2.1.

Reply:
ooh that's a fun one. can you repro it?

Reply:
The Roslyn libraries are dependent on `System.Dynamic.Runtime` which is in turn dependent on `System.Reflection.Emit.ILGeneration`. Paket seems to pickup on the transitive dependency and adds a direct reference to both. If I don't use paket and just add a `ProjectReference`, to `System.Dynamic.Runtime` then nuget doesn't pickup the dependency (presumably because it declares it's dependency on `System.Reflection.Emit.ILGeneration` as `exclude="Compile"`).